src/appimagetool.c

```diff
@@ -131,13 +131,26 @@ int sfs_ls(char* image) {
 * execlp(), execvp(), and execvpe() search on the $PATH */
 int sfs_mksquashfs(char *source, char *destination, int offset) {
     pid_t pid = fork();
-    
     if (pid == -1) {
-        // error, failed to fork()
+        perror("fork()");
```

Comment:
Please add some context information. We use other subprocesses, too, within the scope of AppImageKit. Ideally, the message informs the user that this was trying to open the `mksquashfs` process.